stable/atlantis/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.8.2"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.6.2
+version: 3.7.2
```

Comment:
when bumping a minor version, the patch version should be reset to 0
so in this case: `3.7.0`

----
https://semver.org/#spec-item-7
the same would be for major version, so if this were a PR with major changes, then it'd be `4.0.0`

Reply:
nice catch 🍺